check.go

```diff
@@ -561,11 +562,15 @@ func newSuiteRunner(suite interface{}, runConf *RunConf) *suiteRunner {
 	if conf.Stream {
 		verbosity = 2
 	}
+
+	if conf.Reporter == nil {
```

Comment:
Could we add a unit test for this behaviour? It seems that we can check it from the outside